suricata/update/main.py

```diff
@@ -50,18 +50,20 @@
     sys.path.insert(
         0, os.path.abspath(os.path.join(__file__, "..", "..", "..")))
 
-import suricata.update.rule
-import suricata.update.engine
-import suricata.update.net
-import suricata.update.loghandler
-from suricata.update import config
-from suricata.update import configs
-from suricata.update import extract
-from suricata.update import util
-from suricata.update import sources
-from suricata.update import commands
-from suricata.update import exceptions
-from suricata.update import notes
+from suricata.update import (
+    commands,
+    config,
+    configs,
+    engine,
+    exceptions,
+    extract,
+    loghandler,
+    net,
+    notes,
+    rule as rule_mod,
+    sources,
+    util,
+)
```

Comment:
@vagishagupta23 I'm sorry I did not check the following in the last PR. I used a few tools to achieve this and they do not put the modules on separate lines and just follow the 79 char limit. Is there a PEP8 standard for doing this?

Reply:
@shivan1b There is nothing specifically mentioned in PEP8. However, an example is provided that parentheses is permitted to enclose multiple module names but modules there are not put on separate lines. Here is a link for that https://www.python.org/dev/peps/pep-0328/#id7.
I think writing module names on separate line makes it more readable.
Shall I also follow the 79 char limit?

Reply:
hmm. @jasonish what do you think looks better?

Reply:
Looks wise it seems fine.  But a lot of these are now names commonly used for vars and what not, so I'd worry that vars may shadow import too often?  Probably not an issue after main.py is broken out into more files though.

Reply:
@vagishagupta23 could you please make sure that these import names are not used as variable names? Once done, ping me. I shall recheck and approve.

Reply:
@shivan1b Sure. I will let you know.

Reply:
@shivan1b I checked whether the import names are used as variable names and found 
only the `rule` module having the conflict.
For that, I have created an alias `rule_mod` in the PR. Rest of the import
names are not used as the variable names in main.py.
Could you please recheck and tell if any changes are required?

Reply:
LGTM. Thanks!